File: integration-tests/hibernate-orm-data/src/main/java/io/quarkus/it/hibernate/processor/data/puother/MyOtherEntity.java
package io.quarkus.it.hibernate.processor.data.puother;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.Id;

@Entity
public class MyOtherEntity {

    @Id
    @GeneratedValue
    public Integer id;

    @Column(unique = true)
    public String name;

    MyOtherEntity() {
    }

    public MyOtherEntity(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return "MyOrmEntity [id=" + id + ", name=" + name + "]";
    }

}

File: extensions/panache/hibernate-reactive-panache-kotlin/deployment/src/test/kotlin/io/quarkus/hibernate/reactive/panache/kotlin/test/MyOtherEntity.kt
package io.quarkus.hibernate.reactive.panache.kotlin.test

import io.quarkus.hibernate.reactive.panache.kotlin.PanacheCompanion
import io.quarkus.hibernate.reactive.panache.kotlin.PanacheEntity
import javax.persistence.Entity

@Entity
class MyOtherEntity : PanacheEntity() {
   companion object : PanacheCompanion<MyOtherEntity>
   var name: String? = null
}